tools/mason/MasonPublish.chpl

```diff
@@ -250,24 +307,37 @@ proc getPackageName() throws {
 /* Adds package to the Bricks of the mason-registry branch and then adds the version.toml
  with the source url of the package's GitHub repo.
  */
-private proc addPackageToBricks(projectLocal: string, safeDir: string, name : string) throws {
+private proc addPackageToBricks(projectLocal: string, safeDir: string, name : string,registryPath : string, isLocal : bool) throws {
   try! {
-
     const toParse = open(projectLocal+ "/Mason.toml", iomode.r);
-    const url = gitUrl();
     var tomlFile = new owned(parseToml(toParse));
     const versionNum = tomlFile['brick']['version'].s;
```

Comment:
If version is not present in the ``mason.toml`` the user us greeted with a ``./MasonPublish.chpl:314: error: attempt to dereference nil``. The same goes for the ``getPackageName`` function. this may be a more systemic problem of Mason itself however. If this cannot be fixed here easily and is more of a systemic problem, make sure to open an issue about it.

Reply:
After experimenting, Im thinking this is a more underlying mason issue. Ill open an issue about it.